docs/niswitch/class.rst

```diff
@@ -1183,15 +1183,15 @@ max_carry_dc_current
 
 
 
-        .. tip:: This property can use repeated capabilities (usually channels). If set or get directly on the
-            max_carry_dc_current.Session object, then the set/get will use all repeated capabilities in the session.
+        .. tip:: This property can use repeated capabilities (channels). If set or get directly on the
+            niswitch.Session object, then the set/get will use all repeated capabilities in the session.
             You can specify a subset of repeated capabilities using the Python index notation on an
-            max_carry_dc_current.Session instance, and calling set/get value on the result.:
+            niswitch.Session repeated capabilities container, and calling set/get value on the result.:
 
             .. code:: python
 
-                session['0,1'].max_carry_dc_current = var
-                var = session['0,1'].max_carry_dc_current
+                session.channels[0,1].max_carry_dc_current = var
```

Comment:
On `niswitch`, it's pretty rare for channels to be 0,1... usually they are c0, r1, com2.
Most of these would error if called with two channels.

Reply:
Furthermore, a lot of these are read-only properties.
I think we should merge this PR but open an issue about this additional problem.

Reply:
I do not know how to fix that, unless we add even more metadata somewhere.

Reply:
True. I was thinking of the `0,1` part. I can fix the r/w/rw pretty easy.

Reply:
Add r/w/rw specific tips